tests/cross/src/test/scala/tests/pc/HKSignatureHelpSuite.scala

```diff
@@ -30,4 +30,43 @@ class HKSignatureHelpSuite extends BaseSignatureHelpSuite {
     ),
   )
 
+  // https://github.com/scalameta/metals/issues/5055
+  check(
+    "named".tag(IgnoreScala3),
+    """|
+       |object demo {
+       |  val f: Int = fun(
+       |    logHeaders = true,
+       |    logBody = true,
+       |    logAction = None,
+       |    @@
+       |  )
+       |  def defaultRedactHeadersWhen(name: String): Boolean = false
+       |  
+       |  /**
+       |    * 
+       |    *
+       |    * @param logHeaders
+       |    * @param logBody
+       |    * @param redactHeadersWhen test description
+       |    * @param logAction
+       |    * @return
+       |    */
+       |  def fun(
+       |    logHeaders: Boolean,
+       |    logBody: Boolean,
+       |    redactHeadersWhen: String => Boolean = defaultRedactHeadersWhen,
+       |    logAction: Option[String => Unit] = None,
+       |  ): Int = ???
+       |}
+       |""".stripMargin,
+    """|**Parameters**
+       |- `redactHeadersWhen`: test description
+       |fun(<logHeaders: Boolean>, <logBody: Boolean>, <logAction: Option[String => Unit] = None>, <redactHeadersWhen: String => Boolean = defaultRedactHeadersWhen>): Int
+       |                                                                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+       |  @param <redactHeadersWhen test description
```

Comment:
What's up with this, why is there no closing ">"? Does it just cut off, 'cause it's too long?

Reply:
That's just in the test, there is a regex that cuts off after first space of the label, don't remember why it's there but this is only in the tests. I can try to remove the `<` but it's not an issue for the users